src/components/Card/Card.jsx

```diff
@@ -1,6 +1,6 @@
-import React from "react";
-
+import "./Card.css";
 export const Card = ({ products, featured }) => {
```

Comment:
you can rename this as productsWrapper or productsList or something similar.

Reply:
Nice suggestion, will do that.